src/seeds.proposals.cpp

```diff
@@ -382,6 +419,10 @@ void proposals::onperiod() {
             burn(pitr->staked);
           }
 
+          if (pitr->campaign_type == campaign_invite_type) {
+            send_return_funds_campaign(pitr->campaign_id);
+          }
+
```

Comment:
I guess invite campaigns need to stay in evaluate until all their money is spent?!